.github/build.in.yml

```
@@ -534,6 +537,13 @@ jobs:
           # Only return if PR is still open
```

Apologies if you already tested this and it didn't work, but [the readme of that action](https://github.com/8BitJonny/gh-get-current-pr?tab=readme-ov-file#pull_request-trigger) suggests the following for `pull_request` (and presumably `pull_request_target`:
```suggestion
          sha: ${{ github.event.pull_request.head.sha }}
          # Only return if PR is still open
```

---

Ah, wait that was your first commit.

---

I'm happy with your workaround once it passes your tests, but I've opened an issue in 8BitJonny/gh-get-current-pr for this (236 to avoid backlinking to this PR); maybe we can get it fixed upstream at some point.